doc/news/changes/minor/20160214MathiasMentler

```diff
@@ -0,0 +1,4 @@
+Fixed: double contraction of two SymmetricTensors<..,VectorizedArray<T>> now works. Introduced NumberType<T> with static member ::value to be called instead of the constructor Number() in symmetric_tensor.h.
```

Comment:
nitpick, would you please break this line into several, somewhere around 80 char long?